### ISSUE TITLE
Trying to convert weights into TF - dim problem

### ISSUE THREAD

**Issue by tomerkeren42** [BlcaKHat/yolov3-Helmet-Detection#13]
Hi, and thanks for uploading those files.
due to problem using darknet directly, I'm using a tf translation for the dark net.
it has a convertor - from weights file to .tf file, so the net could use.

Your weight file (and another one I got from kaggle), I got dim problem - can't reshape an array of size 4607 into shape (18,256,1,1) (which is 4608 parameters). 

Do you have any idea what could be the source of the problem?
Other weight files I use are doing just fine.

Thank you for answering, 
Tomer

**Comment by BlcaKHat** [comment]
looks like there is an issue with the array size. i haven't converted it before. can you show me related code >

**Comment by tomerkeren42** [comment]
happens in both torch - conversion and TF conversion.
attaching pytorch code:
![image](https://user-images.githubusercontent.com/62999166/99178751-0a95f000-271f-11eb-981e-db8c5aa08646.png)
![image](https://user-images.githubusercontent.com/62999166/99178757-17b2df00-271f-11eb-940e-564f8cf05d56.png)
keeps getting bad shape at line 322
attaching error:
![image](https://user-images.githubusercontent.com/62999166/99178761-28fbeb80-271f-11eb-968f-91445da119aa.png)

thanks for help!

**Comment by BlcaKHat** [comment]
https://github.com/mystic123/tensorflow-yolo-v3/issues/62
try reading this. it might help.

**Comment by BlcaKHat** [comment]
https://github.com/PINTO0309/OpenVINO-YoloV3/issues/15
this one as well.

**Comment by tomerkeren42** [comment]
thanks!
the problem was changing between 32bit and 64 system.
solution was changing this line:
            header = np.fromfile(f, dtype=np.int32, count=4)  # First five are header values
to :
            header = np.fromfile(f, dtype=np.int32, count=4)  # First five are header values

thanks.

**Comment by BlcaKHat** [comment]
can you share converted files? just curious how it behaves.

**Comment by tomerkeren42** [comment]
first one is this
https://github.com/zzh8829/yolov3-tf2/blob/master/convert.py
second one (which i was finnaly using) is this 
https://github.com/eriklindernoren/PyTorch-YOLOv3/blob/master/detect.py

thanks alot friend.
